tests/PhpGenerator/Helpers.dump().phpt

```diff
@@ -39,6 +39,12 @@ Assert::same('[]', Helpers::dump([]));
 Assert::same('[$s]', Helpers::dump([new PhpLiteral('$s')]));
 
 Assert::same('[1, 2, 3]', Helpers::dump([1, 2, 3]));
+Assert::same('[1, 2, 3]', Helpers::dump([1, 2, 3], 91));
```

Comment:
Length of the `[1, 2, 3]` is *9* + $initLength =*91* is *100* in total and this should fit into the one line.